docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -169,4 +445,111 @@ TBD
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use JSON Schema file ``schema.jsonschema`` with content
+
+.. code:: sh
+  {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+      "values": {
+        "type": "object",
+        "properties": {
+          "greeting": {
+            "type": "string"
+          },
+          "recipient": {
+            "type": "string"
+          }
+        },
+        "required": ["greeting", "recipient"],
+        "additionalProperties": false
+      }
+    },
+    "required": ["values"],
+    "additionalProperties": false
+  }
+
+and YAML file ``values.yaml`` with content
+
+.. code:: sh
+
+  values:
+    greeting: Hello
+    recipient: World
+
+* Validate a YAML config against a given JSON schema:
+
+  .. code:: sh
+
+    $ uw config validate --schema-file schema.jsonschema --input-file values.yaml
+    [2024-01-03T17:23:07]     INFO 0 schema-validation errors found
+
+  Shell redirection via ``|``, ``>``, et al may also be used to stream output to a file, another process, etc.
+
+
+* Read the config from ``stdin`` and render to ``stdout``:
+
+  .. code:: sh
+
+    $ cat values.yaml | uw config validate --schema-file schema.jsonschema
+    [2024-01-03T17:26:29]     INFO 0 schema-validation errors found
+
+
+* However, you cannot read the schema from ``stdin`` and render to ``stdout``:
```

Comment:
As a general rule for documentation -- or at least as a style guideline for `uwtools` -- I'd recommend avoiding pronouns like "we", "you", etc. It's more straightforward to just describe the software and what it does. There may be more room for pronouns in the breezier style of an intro page, but in the drier technical docs, I personally find it distracting. This is just my opinion, though, so I'd like to hear what others think.

For example, this could read
> However, reading the schema from `stdin` is not supported:

(I left out the "render" part, as this is "validate" mode.)

There's another "you" in the `realize` section that could go if we agree on this style point.

Reply:
Bad conversational habit drifting in, thanks for catching that. I've fixed both entries I think

Reply:
Hmmmm. I typically write in a similar conversational style. In a few places in my work-in-progress, I've made a change to remove the pronouns, but what about:

> We recommend a descriptive one to make it easier to read.

Here, I feel like it makes sense to keep it a bit more personal. Thoughts?

Reply:
I don't think "we recommend" adds anything over
> A descriptive [thing] is easier to read.

Good advice doesn't need to be softened. We're all adults here. That "we" recommend an idea isn't what makes it good. The facts should be persuasive on their own.

Setting aside that I don't know the context to be more specific than "thing", in general that sentence sounds weak to me -- "one" and "it" are vague, even if the context clarifies what they refer to. For example, instead of writing (I'm just making this up)
> You can define a variable here to frobnicate the foo. We recommend a descriptive one to make it easier to read.

could be

> Define a variable to frobnicate the foo. A descriptive variable name will clarify its intent to readers.

A strong statement followed by clear rationale. Or even just

> Define a descriptively named variable to frobnicate the foo.

Do we need to explain why descriptive variables are desirable? Attention is a scarce resource and, when I'm the reader, I appreciate documentation that gets to the point.

That said, I think "We" can still be appropriate if substituting the "The UW Team" or "The UFS Community" or whatever is really what we mean and the context is clear. That's different, though, from e.g. "We see no output when we use the `--quiet` flag" (who is "we"?), which can be stated more strongly/directly as "The `--quiet` flag suppresses all output".

Reply:
Point taken...with a few laughs thrown in for good measure. "Frobnicate the foo"?!? 😆

Reply:
https://en.wiktionary.org/wiki/frobnicate

Reply:
TIL.